app/lib/common/dialogs/nuke_confirmation.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:acter/features/onboarding/providers/onboarding_providers.dart';
+import 'package:atlas_icons/atlas_icons.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:go_router/go_router.dart';
+
+// Can be extended to be reusable dialog as riverpod states get added.
+// Ref can be used to read any provider which are declared.
+void nukeConfirmationDialog(BuildContext context, WidgetRef ref) {
+  showDialog(
+    context: context,
+    builder: (BuildContext ctx) {
+      return AlertDialog(
+        title: const Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Icon(Atlas.bomb_bold,color: Colors.red,),
+            SizedBox(height:10,),
+            Text('Nuke all local data'),
+          ],
+        ),
+        content: RichText(
+          textAlign: TextAlign.left,
+          text: const TextSpan(
+            text: 'Attention: ',
+            style: TextStyle(color: Colors.white, fontSize: 15),
+            children: <TextSpan>[
+              TextSpan(
+                text:
+                    'Nuking removes all local data, including encryption keys. If this is your last signed-in device you might no be able to decrypt any previous content.',
+              ),
+              TextSpan(text: 'Are you sure you want to nuke?'),
```

Comment:
Nuke seems a strong wording. I would probably go for ```Erase``` or ```Wipe``` in terms of easy understanding and no assumed double meanings.

Reply:
It is intentionally strong wording as this is super dangerous... and should only taken if there is no other course of action left ... it won't even back up ... Hence also using the bomb_icon and big-red-letters ...

I mean ... _ideally_ this screen never shows up to a regular user. It only does when we got the underlying SDK into an unrecoverable state - most likely upon startup.